src/structures/Interaction.js

```diff
@@ -105,6 +105,14 @@ class Interaction extends Base {
     return this.client.guilds.cache.get(this.guildID) ?? null;
   }
 
+  /**
+   * Indicates whether this interaction is received from a guild.
+   * @returns {boolean}
+   */
+  inGuild() {
+    return this.guildID && this.member;
```

Comment:
```suggestion
    return Boolean(this.guildID && this.member);
```
so it actually returns a boolean